From a32d153809f10f46c1cd385043e4f5e1b2d107ef Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Sun, 2 Oct 2016 22:08:47 +0000
Subject: [PATCH] OS-5695 bunyan, mergeq, workq tests fail to run by default
 Reviewed by: Cody Mello <melloc@joyent.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Reviewed by: Jerry Jelinek
 <jerry.jelinek@joyent.com> Approved by: Jerry Jelinek
 <jerry.jelinek@joyent.com>

---
 usr/src/test/util-tests/runfiles/default.run | 6 +++---
 usr/src/test/util-tests/tests/Makefile       | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/usr/src/test/util-tests/runfiles/default.run b/usr/src/test/util-tests/runfiles/default.run
index dcdbf43f95..f0d8c90045 100644
--- a/usr/src/test/util-tests/runfiles/default.run
+++ b/usr/src/test/util-tests/runfiles/default.run
@@ -25,12 +25,12 @@ outputdir = /var/tmp/test_results
 [/opt/util-tests/tests/printf_test]
 [/opt/util-tests/tests/allowed-ips]
 [/opt/util-tests/tests/vnic-mtu]
-[/opt/util-tests/tests/bunyan]
+[/opt/util-tests/tests/bunyan/bunyan]
 
 [/opt/util-tests/tests/xargs_test]
 
-[/opt/util-tests/tests/mqt]
-[/opt/util-tests/tests/wqt]
+[/opt/util-tests/tests/mergeq/mqt]
+[/opt/util-tests/tests/mergeq/wqt]
 
 [/opt/util-tests/tests/libnvpair_json]
 tests = ['json_00_blank', 'json_01_boolean', 'json_02_numbers',
diff --git a/usr/src/test/util-tests/tests/Makefile b/usr/src/test/util-tests/tests/Makefile
index 739b79893f..452ea1400c 100644
--- a/usr/src/test/util-tests/tests/Makefile
+++ b/usr/src/test/util-tests/tests/Makefile
@@ -14,6 +14,6 @@
 # Copyright 2014 Garrett D'Amore <garrett@damore.org>
 #
 
-SUBDIRS = dladm libnvpair_json printf xargs mergeq workq
+SUBDIRS = dladm libnvpair_json printf xargs bunyan mergeq workq
 
 include $(SRC)/test/Makefile.com
-- 
2.21.0

